cmd/integrationArtifactUpload_test.go

```diff
@@ -165,6 +165,6 @@ func TestRunIntegrationArtifactUpload(t *testing.T) {
 		httpClient := httpMockCpis{CPIFunction: "", ResponseBody: ``, TestType: "NegativeAndUpdateIntegrationDesigntimeArtifactResBody"}
 
 		err = runIntegrationArtifactUpload(&config, nil, &filesMock, &httpClient)
-		assert.EqualError(t, err, "HTTP POST request to https://demo/api/v1/IntegrationDesigntimeArtifacts failed with error: []: Internal error")
+		assert.EqualError(t, err, "HTTP POST request to https://demo/api/v1/IntegrationDesigntimeArtifacts failed with error: : Internal error")
```

Comment:
Would it be possible to add a real http status to this mock error message? Such as 'Unauthorized'

Reply:
done